lib/money/money.rb

```diff
@@ -190,10 +191,10 @@ def to_s(style = nil)
     when :legacy_dollars, nil
       sprintf("%.2f", value)
     when :legacy_cents
-      (value * 100).round.to_i.to_s
-    when :minor_units
-      (value * currency.subunit_to_unit).round.to_i.to_s
-    when :major_units
```

Comment:
They were never used, it's something we added in the currency PR. Just renaming for consistency